theano/gpuarray/opt.py

```diff
@@ -106,6 +105,11 @@ def query(self, *tags, **kwtags):
 gpu_seqopt.register('gpuarray_cut_transfers', gpu_cut_copies, 2,
                     'fast_compile', 'fast_run', 'gpuarray')
 
+gpu_seqopt.register('op_lifter_topo', TopoOptimizer(gpu_topo.query('+fast_compile', '+fast_run', '+cudnn',
+                                                                   '+cudnn', '+conv_dnn', '+gpuarray'),
+                                                    order='out_to_in'),
+                    0.5, 'fast_run', 'gpuarray', 'fast_compile')
+
```

Comment:
remove this and the line 80. It isn't used for now as nothing is registered in gpu_topo. We don't have benchmark that show this would be useful, so I would not finish this until we see a need for it.